### ISSUE TITLE
Make examples easier to access

### ISSUE THREAD

**Issue by Ellzord** [threadly/threadly#145]
First off this isn't a bug so you can mark it with Question or whatever you deem appropriate.

It took a bit of digging to find your examples and then they weren't too helpful either. My suggestion would be:
* Create a side bar for your wiki (easier to navigate)
* Link to your examples on the front page
* More than just method doc on your examples - for a first time user I wana know what some of the code blocks do
* See [GitHub markdown](https://help.github.com/articles/github-flavored-markdown/) and provide some snippets in the wiki. Cool little pieces of code as well as full examples. This way you can attract people to your functionality and then they can see it in full swing in the other repo.

Right now it feels like I have to invest a lot of my own time to see what your framework can do for me.

Just my thoughts

**Comment by jentfoo** [comment]
Thank you for your issue!  I definitely want to make this library easier to understand and dive into.  Part of my struggle is that there is so much it can provide, I don't want to drown people in information.  Originally threadly examples was just designed to be some complete apps people can run to see something running it.  Now in many ways our socket library litesockets and our new project ambush are even better examples of using it.

Did you by chance find the features wiki page? https://github.com/threadly/threadly/wiki/Threadly-Features

Maybe it would make sense to make that more visible.  And possibly make a wiki page for each feature.  In the page we could provide not just a textual description but code examples as well.  Do you think that would be easier to find and digest?  Or do you think it would be better to add small code examples to our examples repo?

**Comment by Ellzord** [comment]
I didn't actually see that page but I think a mix of the two would be good.

If you have a feature you think is cool, it may sell the library for you, so you should write about it as you have but also provide some snippets and a link to the example its being used. Someone isn't going to navigate through loads of source files to find the bit you want them to - so show them why it's cool and then they'll want to find it :+1:

**Comment by jentfoo** [comment]
I agree.  As you probably know the documentation part is not the most fun for any project, but it absolutely is important.  I will keep this issue open to indicate expanding and improving in that area everything you described above.

Thanks for the feedback, and if you think of anything else (or have any questions), don't hesitate to open an issue.

**Comment by Ellzord** [comment]
True that. Good luck!
